.flake8

```diff
@@ -0,0 +1,13 @@
+[flake8]
+max-line-length = 119
```

Comment:
Maybe we should align this value with `line-length` in pyproject.toml? Or is this value higher on purpose to give some leeway?

Reply:
We can. We didn't agree on the line length yet, so I didn't pay attention to it.